### ISSUE TITLE
install issue

### ISSUE THREAD

**Issue by nsteinme** [cortex-lab/alyx#529]
Trying to follow the instructions verbatim on an amazon ubuntu 18.04 instance, I get:
```
(alyxvenv) ubuntu@ip-172-31-27-221:~/alyx$ pip install -r requirements.txt
Collecting git+https://github.com/Python-Markdown/markdown (from -r requirements.txt (line 17))
  Cloning https://github.com/Python-Markdown/markdown to /tmp/pip-req-build-_dcshexi
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  Complete output from command /home/ubuntu/alyx/alyxvenv/bin/python3 /home/ubuntu/alyx/alyxvenv/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpd31j0hx2:
  Traceback (most recent call last):
    File "/home/ubuntu/alyx/alyxvenv/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 207, in <module>
      main()
    File "/home/ubuntu/alyx/alyxvenv/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 197, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/ubuntu/alyx/alyxvenv/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py", line 54, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-ivwxdh4t/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 130, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-ivwxdh4t/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 112, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-ivwxdh4t/overlay/lib/python3.6/site-packages/setuptools/build_meta.py", line 126, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 26, in <module>
      from markdown import __version__, __version_info__
  ModuleNotFoundError: No module named 'markdown'

  ----------------------------------------
Command "/home/ubuntu/alyx/alyxvenv/bin/python3 /home/ubuntu/alyx/alyxvenv/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpd31j0hx2" failed with error code 1 in /tmp/pip-req-build-_dcshexi
```

The markdown git repository's setup.py indeed tries to import from markdown, which does seem like a loop to me, but I'm not sure how these things normally work? Probably I did something else wrong?

**Comment by oliche** [comment]
Fixed by 05b0f58cfae668616038d1647fa4097370c70838.
You can either wait Monday for the release, checkout dev or cherry-pick commit as it's a fairly simple fix on the requirement file.

**Comment by nsteinme** [comment]
thanks!!
